[FEATURE] Affichage de l'onglet "Activité" pour les campagnes de collecte de profils (PIX-2772).

lisequesnel opened the pull request:
## :unicorn: Problème
Jusqu'alors l'onglet "Activité" n'était disponible que pour les campagnes d'évaluation.

## :robot: Solution
Ouvrir ce dernier pour les campagnes de collecte de profils.

## :rainbow: Remarques
BSR : Prise en compte des participations multiples (récupération de la dernière participation seulement)

## :100: Pour tester
- Se connecter à Pix Orga
- Choisir une campagne de collecte
- Vérifier qu'on arrive sur l'onglet "Activité"
- Cliquer sur l'activité d'un participant
- Vérifier qu'on arrive sur le détail de ce participant

BSR :
- Pour les campagnes à envois multiples, dans la liste des campagnes, le chiffre a changé + dans l'onglet Résultats des 2 types de campagnes

lisequesnel committed:
[ORGA] Display activity tab for profiles collection campaigns

Commit:
[ORGA] change tab name for profiles collection campaign

lisequesnel committed:
[API] Filter only shared campaignProfilesCollectionParticipationSummaries

Commit:
[API] change request to display only campaignParticipations not improved

lisequesnel committed:
[API] Filter only not improved participations for get campaignReport

Commit:
[API] Filter only not improved participations for findPaginatedFilteredByOrganizationId in campaignReport

lisequesnel committed:
[ORGA] Redirect to right page when click on participant activity

based uppon campaign type

pix-service commented:
I'm deploying this PR to these urls:

- App (.fr): https://app-pr3234.review.pix.fr
- App (.org): https://app-pr3234.review.pix.org
- Orga: https://orga-pr3234.review.pix.fr
- Certif: https://certif-pr3234.review.pix.fr
- Admin: https://admin-pr3234.review.pix.fr
- API: https://api-pr3234.review.pix.fr/api/

 Please check it out!

lisequesnel commented on orga/app/components/routes/authenticated/campaign/report.hbs:
```diff
@@ -59,20 +59,15 @@
-          @model={{@campaign}}
-          class="navbar-item"
-        >
-          {{t 'pages.campaign.tab.participants' count=@campaign.participationsCount}}
```
**suggestion** : enlever les trad

lisequesnel commented on api/tests/integration/infrastructure/repositories/campaign-profiles-collection-participation-summary-repository_test.js:
```diff
@@ -130,6 +130,28 @@ describe('Integration | Repository | Campaign Profiles Collection Participation
+        const results = await campaignProfilesCollectionParticipationSummaryRepository.findPaginatedByCampaignId(campaignId);
+
+        // then
+        expect(results.data[0].id).to.equal(recentCampaignParticipation.id);
```
**suggestion** : rajouter `expect(results.data).to.have.lengthOf(1)`

lisequesnel commented:
func review en mob :)

lisequesnel committed:
[ORGA] Return to activity page when back on profile page

this behaviour is not ideal, but at least it is the same as assessment campaigns
